packages/cli/src/utils/getUrlListFromArgs.ts

```diff
@@ -59,95 +64,118 @@ const parseUrlsFromCSV = async (csvPath: string, spinnies: any) => {
     }, [] as string[]);
 
     if (_urls.length === 0) {
-      console.log(`Error: CSV file contains no URLs: ${basename(csvPath)}`);
+      console.log(
```

Comment:
This can be converted into a logger function inside the same file, as it's used in many places.